biosimulations/apps/dispatch/src/app/components/simulations/view/view.component.ts

```diff
@@ -86,26 +86,29 @@ export class ViewComponent implements OnInit {
           this.outLog = data.message;
           this.errLog = '';
         } else {
-          const out = data.data.output;
-          const err = data.data.error;
-
-          if (err !== "") {
-            this.outLog = 'Log not available';
-            this.errLog = err;
-          } else {
-            this.outLog = out;
-            this.errLog = '';
-          }
+          this.outLog = data.data.output;
+          this.errLog = data.data.error;
+
+          // if (err !== "") {
```

Comment:
This commented out code (L92-98) can be remove

Reply:
Done